nmma/tests/injections.py

```diff
@@ -59,6 +59,8 @@ def create_injection_from_command_line(model_name):
 
         if model_name == "nugent-hyper":
             prior_path = os.path.join( priorDir, "sncosmo-generic" + ".prior")
+        elif model_name == "TrPi2018":
+            prior_path = os.path.join( dataDir, "constTrPi2018" + ".prior") #pinning the parameter svalues in the prior file
```

Comment:
Can we call this TrPi2018_pinned_parameters or similar?

Reply:
Done

Reply:
@sahiljhawar TrPi2018_pinned_parameters.prior, you are missing the period.

Reply:
Sorry, my bad!